MatrixSDK/Crypto/Algorithms/MXEncrypting.h

```diff
@@ -60,7 +60,7 @@
 
  @return a MXHTTPOperation instance. May be nil if all required materials is already in place.
  */
-- (MXHTTPOperation*)ensureSessionForUsers:(NSArray<NSString*>*)users
+- (MXHTTPOperation*)ensureSessionForUsers:(NSArray<NSString*>*)users forceDistributeToUnverified: (BOOL) forceDistributeToUnverified
```

Comment:
Sadly we don't have obj-c style checker / formatter to check this automatically, but to aligh the style:
- no spaces between type and parameter, i.e. `forceDistributeToUnverified: (BOOL) forceDistributeToUnverified` => `forceDistributeToUnverified:(BOOL)forceDistributeToUnverified` (across the whole PR)
- if some parameters are on multiple lines (e.g. `success` and `failure`), then all should be, incl `forceDistributeToUnverified`, and aligned vertically by the double colon